ironfish/src/wallet/walletdb/accountValue.ts

```diff
@@ -56,6 +63,7 @@ export class AccountValueEncoding implements IDatabaseEncoding<AccountValue> {
     size += KEY_LENGTH
     size += KEY_LENGTH
     size += PUBLIC_ADDRESS_LENGTH
+    size += 1 // version
```

Comment:
@ anyone : where would be a good place to define `VERSION_LENGTH`? Just at the top of `accountValue.ts` here?

Reply:
doing that